package.json

```
@@ -0,0 +1,33 @@
+{
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "test:unit": "vitest --environment jsdom --root src/",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
+    "lint": "eslint . --ext .vue,.js,.cjs,.ts --fix --ignore-path .gitignore"
```

Поскольку Prettier был установлен с инициализацией приложения, не лишним будет добавить команду для его запуска. И заодно сразу выполнить её. Это исправит много неточностей в стиле кода, которые есть на текущий момент.

```suggestion
    "lint": "eslint . --ext .vue,.js,.cjs,.ts --fix --ignore-path .gitignore",
    "format": "prettier . --write"
```

---

@nelkor  можем добавить прекоммит-хуки в git, и перед пушем он проверит стили и отформатирует

---

@VladislavBobyrev, да, было бы неплохо)